pkg/apis/kops/containerdconfig.go

```diff
@@ -24,6 +24,9 @@ import (
 // NvidiaDefaultDriverPackage is the nvidia driver default version
 const NvidiaDefaultDriverPackage = "nvidia-driver-535-server"
 
+// NvidiaDevicePluginImage is the Nvidia K8s device plugin container image
+const NvidiaDevicePluginImage = "nvcr.io/nvidia/k8s-device-plugin:v0.17.3"
```

Comment:
Let's group these in a const block.
```suggestion
const(
    // NvidiaDefaultDriverPackage is the nvidia driver default version
    NvidiaDefaultDriverPackage = "nvidia-driver-535-server"
    // NvidiaDevicePluginImage is the Nvidia K8s device plugin container image
    NvidiaDevicePluginImage = "nvcr.io/nvidia/k8s-device-plugin:v0.17.3"
)
```